importexport/musicxml/exportxml.cpp

```diff
@@ -3234,11 +3234,12 @@ static void writePitch(XmlWriter& xml, const Note* const note, const bool useDru
       if (fabs(tuning) > 0.01) {
             alter2 = tuning / 100.0;
       }
-      if (alter && !alter2)
-            xml.tag("alter", alter);
-      if (!alter && alter2)
-            xml.tag("alter", alter2);
-      // TODO what if both alter and alter2 are present? For Example: playing with transposing instruments
+      // `alter` represents the "regular" (Western) pitch which can be 
+      // 0 (natural), 1 (sharp), -1 (flap) or some other integer depending on transposing instruments.
+      // `alter2` represents a microtone or manually-adjusted note tuning.
+      // In MusicXML, These two values are merged in the same "alter" tag.
+      if (alter || alter2)
+            xml.tag("alter", (double)alter + alter2);
```

Comment:
Not `double(alter+alter2)`, i.e. casting the result of the addition rather than just one of the 2 and in a more C++ way?

Reply:
Not sure, one of them is int and the other double.

Reply:
OK, I see, but the result is a double in any case.